lib/Viroverse/Controller/Sample.pm

```diff
@@ -160,6 +163,23 @@ sub ice_cultures : Chained('page_base') PathPart('ice-cultures') Args(0) {
     $c->detach( $c->view("NG") );
 }
 
+sub isla_worksheet : Chained('page_base') PathPart('isla-worksheet') Args(0) {
```

Comment:
Chain this off `load` rather than `page_base`; `page_base` does extra work to prepare a `Viroverse::SampleTree` which we don't require here.